.github/workflows/maven.yml

```diff
@@ -21,6 +21,7 @@ on:
   pull_request:
```

Comment:
Lets update the comment to match this change.

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/102/commits/e400f18a84683f52997137985edffd9179622dd2